tests/mce_check/mce_check.cpp

```diff
@@ -78,6 +78,13 @@ static int mce_check_run(struct test *test, int cpu)
 
     if (errorcount)
         log_platform_message(SANDSTONE_LOG_ERROR "MCE interrupts detected on %d CPUs", errorcount);
+
+    uint64_t thermal_now = sApp->count_thermal_events();
+    if (thermal_now != sApp->last_thermal_event_count) {
+        sApp->last_thermal_event_count = thermal_now;
+        log_platform_message(SANDSTONE_LOG_WARNING "Thermal events detected.");
```

Comment:
Does it make sense to also output the number of events detected? It would give a sense of how severe the overheating was...

Reply:
Well, the number of events tells us little, because the ratio at which they come is not something obvious. But printing the number might help the user knowing which loop of execution those events happened in, so adding that. For MCE too.